### DISCUSSION TITLE
V4.9.1 version identification

### DISCUSSION THREAD

**Discussion by JCV67** [wazuh/wazuh#26392] [General]
Wazuh manager is not recognizing v4.9.1 as being a  bigger version than 4.9.0.

**Comment by MiguelazoDS** [comment]
Hi @JCV67, 

How did you notice that? is there any error you faced? Is it not letting you connect a 4.9.1 agent or something?

**Reply by JCV67** [reply]
Since about 2 hours ago the problem disappeared. What happened is that in Wazuh Dashboard, agents in v4.9.1 where marked as outdated. After some changes in ossec.conf that I made to solve other problems the issue disappeared and I cannot say exactly what solved the problem. What I now can say is that was I problem on my server and not a general issue. Thank you for your interest @MiguelazoDS

**Reply by MiguelazoDS** [reply]
Oh I get it, it's like the manager received the information that the agents are v4.9.1 but still marked them as "old". I don't have in mind how the dashboard obtains that information at this moment, but I'd say that it is just a matter of seconds until the data is synced, similar to "active" and "disconnected", etc. 

Don't hesitate to reach us again, if you find this is a real issue. Regards!
